app/javascript/components/integrations/heroku.vue

```diff
@@ -0,0 +1,42 @@
+<template>
+  <div class="flex flex-row w-1/2 shadow-lg p-7 border-gray-50 rounded-2xl">
+    <div class="w-15">
+      <div class="w-10 border-2 rounded-full mr-7">
+        <img
+          src="https://brand.heroku.com/static/media/heroku-logo-stroke.aa0b53be.svg"
+          class="p-1.5"
+        >
+      </div>
+    </div>
```

Comment:
Esto podría ser un solo div con `w-15` y padding interno o no?

```suggestion
    <div class="w-15 border-2 rounded-full mr-7 p-2.5">
      <img
        src="https://brand.heroku.com/static/media/heroku-logo-stroke.aa0b53be.svg"
        class="p-1.5"
      >
    </div>
```

Reply:
No estoy seguro como, por alguna razón combinarlos en 
`<div class="w-15 px-5 border-2 rounded-full mr-7">`
no funciona. Que hice mal?

Reply:
Jajaja es clásico que las imágenes no se comportan como uno quiere.

Acá te dejo un [tailwind play](https://play.tailwindcss.com/rfEo37m2iR) con algo similar a lo que hacías.

Hay que considerar que si quieres un `p-1.5` en la imagen, es mejor que el contenedor lo tenga y que la imagen se ajuste a las dimensiones del contenedor con `w-full h-full`. 

También definí la altura para que fuese "cuadrado" y en este caso, la imagen también lo es. Si no lo fuese y quieres hacer esa clase de ajustes, con [`object-cover`](https://v2.tailwindcss.com/docs/object-fit#cover) puedes hacer que la imágen cubra toda el área cortando lo necesario y con [`object-contain`](https://v2.tailwindcss.com/docs/object-fit#contain) puedes mostrarla completamente sin deformarla.

Reply:
Entiendo, muchas gracias!